### PR TITLE
updated helm helmfile kubectl

### PR DESCRIPTION
**Pull request by marcelodevops**
I am using the action in a project and we updated our helmfile cli and now we need the latest helmfile version in the action workflow.
I hope it is OK
Best regards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ jobs:
...
 
 ## Optional Inputs
-- `helmfile-version` : helmfile version. Default `"v0.147.0"`.
-- `helm-version` : Helm version. Default `"v3.10.1"`
-- `kubectl-version` : kubectl version. Default `1.23.7`
-- `kubectl-release-date` : kubectl release date. Default `2022-06-29`
+- `helmfile-version` : helmfile version. Default `"v0.148.1"`.
+- `helm-version` : Helm version. Default `"v3.10.2"`
+- `kubectl-version` : kubectl version. Default `1.25.4`
+- `kubectl-release-date` : kubectl release date. Default `2022-11-09`
```
Comment (kondoumh) [inline README.md:35]: Downloading kubectl gives 404 error.
Isn't the latest version 1.24.7?

https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html

**Review by kondoumh — COMMENTED** [review]
You can specify the version in your workflow without changing action.yml.

```yaml
jobs:
  build:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v3
    - name: Setup helmfile
      uses: mamezou-tech/setup-helmfile@v1.1.0
      with:
        helmfile-version: "v0.148.1"
        helm-version: "v3.10.2"
        kubectl-version: "1.24.7"
        kubectl-release-date: "2022-10-31"
```

File: README.md (hunk 1 above)
Comment (marcelodevops) [reply README.md:35]: Hi, I will check and fix the error

Comment (marcelodevops) [reply README.md:35]: @kondoumh Yes, I realised after I created the PR, just trying to contribute

**Review by kondoumh — APPROVED** [review]
LGTMI will merge and release later.

**Comment by kondoumh** [comment]
Released. Thank you.

**Pull request merged**